doc/md/guides/mysql/cd-101.md

```diff
@@ -0,0 +1,156 @@
+---
+id: mysql-cd-101
+title: Detecting drops of foreign-key constraints on MySQL
+slug: /guides/mysql/checks/CD101
+tags: [linting, ci, destructive-change, mysql, drop-foreign-key]
+---
+
+## Introduction
+
+This document describes how Atlas's linter can be used to warn about dropping
+foreign key constraints from a MySQL database.
+
+Let's demonstrate by creating a database with 2 tables - users and pets:
+```sql
+CREATE TABLE users (
+	    id bigint NOT NULL AUTO_INCREMENT,
+	    name varchar(255) NOT NULL,
+	    PRIMARY KEY (id)
+	);
+
+CREATE TABLE pets (
+        id bigint NOT NULL AUTO_INCREMENT,
+        name varchar(255) NOT NULL,
+        owner_id bigint,
+        PRIMARY KEY (id),
+        CONSTRAINT owner_id FOREIGN KEY (owner_id) REFERENCES users (id)
+    );
+```
+
+Since we declared the foreign key constraint on the pets table,
+the database will validate any data inserted to it - blocking insertion
+of rows where the `owener_id` does not correspond to an `id` of an existing user.
+
+Now, let's add a new migration, dropping this constraint as described in the
+[MySQL documentation](https://dev.mysql.com/doc/refman/8.0/en/create-table-foreign-keys.html#:~:text=the%20foreign%20key.-,Dropping%20Foreign%20Key%20Constraints,-You%20can%20drop)
+:
+```sql
+ALTER TABLE pets DROP FOREIGN KEY owner_id;
+```
+
+This action, while won't cause data loss, can have some unpredictable side effects
+(as described below). This is why we may want to run some inspection to provide us 
```

Comment:
```suggestion
(as described below). This is why you may want to run some inspection to provide you 
```

Reply:
do you think I should change "we" to "you" everywhere?